estate/models/estate_property.py

```diff
@@ -0,0 +1,87 @@
+from dateutil.relativedelta import relativedelta
+from odoo import api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.tools.float_utils import float_compare, float_is_zero
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Estate Property"
+
+    name = fields.Char()
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(default=lambda self: fields.Date.today() + relativedelta(months=+3))
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(required=True)
```

Comment:
We haven't covered it on purpose, but for your information, in odoo there is a type of field Monetary, see https://www.odoo.com/documentation/17.0/developer/reference/backend/orm.html#odoo.fields.Monetary